components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.store.ui/src/main/java/org/wso2/carbon/mediator/store/MessageStoreMediator.java

```diff
@@ -135,4 +163,12 @@ private void handleException(String msg) {
         LogFactory.getLog(this.getClass()).error(msg);
         throw new SynapseException(msg);
     }
-}
+
+    public boolean checkForBraces(OMAttribute atr){
+        return ((atr.getAttributeValue().startsWith("{")) && (atr.getAttributeValue().endsWith("}")));
+    }
+
+    public String removeBraces(OMAttribute atr){
+        String newpath = atr.getAttributeValue().substring(1, atr.getAttributeValue().length()-1);
+        return newpath;
+    }}

```

Comment:
Don't we need 'atr.getAttributeValue().trim()' in both checkForBraces() and removeBraces() before checking for braces in case if the attribute value starts with a space.

Reply:
Done